fastseq_cli/transformers_generate.py

```diff
@@ -17,6 +18,37 @@ def chunks(lst, n):
     for i in range(0, len(lst), n):
         yield lst[i:i + n]
 
+def sort_sentences(input_ids: List[str], reverse: bool=False):
+    """Sort the tokenized sentences by the number of tokens.
+
+    Args:
+        input_ids (List[List[int]]): tokenized sentences.
+        reverse (bool): indicate the order is ascending(False) or descending.
+
+    Returns:
+        tuple(List[List[int]], List[int]): the sorted tokenized sentences and
+            the indices in the original input list.
+    """
+    is_ascending = -1 if reverse else 1
+    sorted_idx = sorted(
```

Comment:
Any perf for large data?

Reply:
@yuyan2do  Do you mean the benchmarking result on a larger dataset than the data in our benchmark script?

Reply:
I also have similar worry, as it loaded all data into memory, and then sort. Please check perf on a larger dataset.

Reply:
Here is the test results on a larger dataset

- 5,242,880 of examples (~ 16GB): sort took ~4.4 seconds; unsort took: 1.4 seconds;
- 15,728,640 of examples (~ 48GB): sort took ~14.0 seconds; unsort took: 4.6 seconds;